cmd/rookctl/pool/create_test.go

```diff
@@ -121,7 +125,8 @@ func TestCreatePoolFailure(t *testing.T) {
 		},
 	}
 
-	out, err := createPool("pool1", PoolTypeReplicated, 0, 0, 0, c)
+	pool := model.Pool{Name: "pool1", Type: model.Replicated}
+	out, err := createPool(pool, c)
```

Comment:
perhaps add a test with negative datachunk or coding chunk values?

Reply:
negative values will be ignored just like 0, so we're ok